### DISCUSSION TITLE
Model attributes in Filament models are missing

### DISCUSSION THREAD

**Discussion by FrazeColder** [filamentphp/filament#9772] [Help]
Hey there,

I am just getting started with Filament PHP and have to say, it is such a nice tool! Thank you!

However, I am facing an issue. I have created a simple `UserResource` and started to implement some text columns. However, some of my text columns are empty while they are not being empty in the database. When I dump the current model, I can see that a lot of attributes are missing.

And I am pretty sure I do know why they are missing. Because I am using [Laravel API Resources](https://laravel.com/docs/10.x/eloquent-resources). I can only see exact these attributes I have defined in my API Resources for the given model.

This behavior needs to stay as it is because I don't want to show all the model attributes to the API frontend when people are calling my API. API Resources are typically used to shape the data that is returned when resources are requested through an API. These resources allow you to customize the data that is exposed to the client.

However, I do not understand why Filament PHP is not seeing all the attributes associated with the model. Is there any possibility to turn that behavior off to not use Laravel API Resources within the Filament PHP environment?

Kind regards

**Comment by FrazeColder** [comment]
Update: No, there are also no attributes in the models `$hidden` array set

**Reply by leandrocfe** [reply]
```php
protected $fillable = [...];
```
?
